src/components/LandingPage/LandingPage.tsx

```diff
@@ -11,9 +12,15 @@ export const LandingPage: React.FC = () => {
     <main className="landing">
       <header className="landing-img">
         <h1 className="logo">Backcountry Buddy</h1>
-      {isAuthenticated ?
-        <h2>Welcome, {user.given_name}</h2> :
-        <LoginButton />}
+        
+        {isAuthenticated && user.given_name ? (
```

Comment:
This is good to have as an edgecase of if a user object doesn't come back with an empty name property.